modules/common/SystemInfo.h

```diff
@@ -7,7 +7,12 @@
 #ifndef SYSTEM_INFO_H
 #define SYSTEM_INFO_H
 
-static constexpr double RADIUS = 28.0;  // 車輪の半径[mm]
+#define PI 3.1415926           // 円周率
+#define RAD_TO_DEG 180.0 / PI  // ラジアンを度数に変換するための係数
+#define DEG_TO_RAD PI / 180.0  // 度数をラジアンに変換するための係数
+
 static constexpr double TREAD = 112.0;  // 走行体のトレッド幅（両輪の間の距離）[mm]
+// フォーマットエラー解消のための空行
+static constexpr double WHEEL_RADIUS = 28.0;  // 車輪の半径[mm]
```

Comment:
この行詰めたら勝手にコメントの空行開くの？下みたいにはできそうにない？
```suggestion
static constexpr double TREAD = 112.0;  // 走行体のトレッド幅（両輪の間の距離）[mm]
static constexpr double WHEEL_RADIUS = 28.0;  // 車輪の半径[mm]
```